WordPress/src/test/java/org/wordpress/android/ui/accounts/HelpViewModelTest.kt

```diff
@@ -0,0 +1,46 @@
+package org.wordpress.android.ui.accounts
+
+import androidx.lifecycle.Observer
+import com.nhaarman.mockitokotlin2.times
```

Comment:
These 2 imports should no longer reference `com.nhaarman.mockitokotlin2`, it was removed on `trunk` and merging it will break unit tests for everyone. (Internal ref: p1668591696858459/1668591368.084779-slack-C02QANACA).

Reply:
Thank you for the heads up @ovitrif 🙇 
I'll merge `trunk` before I proceed with testing/merging this PR.

Reply:
Heads up that [only a change in the imports was needed](https://github.com/wordpress-mobile/WordPress-Android/pull/17449/commits/a54f4ea99407d3a6ae7618d15517951178510354) to make the tests work 👍

Reply:
Awesome, thank you so much for resolving the issue @antonis 🙇 🥇 🚀